dotnet-petclinic-payment/PetClinic.PaymentService/appsettings.Development.json

```diff
@@ -0,0 +1,9 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning"
+    }
+  },
+  "PetClinicBaseAddress": "http://ec2-3-106-218-72.ap-southeast-2.compute.amazonaws.com:8080/"
```

Comment:
Why we are using this hardcoded dns name `http://ec2-3-106-218-72.ap-southeast-2.compute.amazonaws.com`?

Reply:
removed